packages/rum-core/src/domain/configuration.ts

```diff
@@ -78,6 +80,13 @@ export function validateAndBuildRumConfiguration(
     }
   }
 
+  if (initConfiguration.excludedActivityUrls !== undefined) {
+    if (!Array.isArray(initConfiguration.excludedActivityUrls)) {
+      display.error('Excluded Activity Urls should be an array')
+      return
+    }
+  }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could spare a `if`
```suggestion
  if (initConfiguration.excludedActivityUrls !== undefined && !Array.isArray(initConfiguration.excludedActivityUrls)) {
    display.error('Excluded Activity Urls should be an array')
    return
  }
```